src/components/form-grid/components/form-column/form-column.stories.js

```diff
@@ -0,0 +1,72 @@
+import { html } from "lit-html";
+import "../../../form";
+import "../../../form-grid";
+import "../../../input-field";
+import "../../../button";
+import "../../../form-message";
+import styles from "../.././styles.scss";
+import formStyling from "../../../form/styles.scss";
+import inputFieldStyling from "../../../input-field/styles.scss";
+import buttonStyling from "../../../button/styles.scss";
+import formMessageStyle from "../../../form-message/styles.scss";
+import { stylesheet, docsIntro } from "../../../../../.storybook/utils.js";
+import { args, argTypes } from "../../../grid/components/column/config";
+
+export default {
+  title: "native-elements/vl-form-grid/vl-form-column",
+  decorators: [
+    (story) =>
+      html`${stylesheet(
+        `${styles}${formStyling}${inputFieldStyling}${buttonStyling}${formMessageStyle}`
+      )}${story()}`,
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          stylesheets: ["form-grid"],
+          root: "form-grid",
+          intro: "Class that represents a column in a form grid layout.",
+        }),
+      },
+    },
+  },
+  args: { ...args },
+  argTypes: { ...argTypes },
+};
+
+export const Default = ({
+  size,
+  maxSize,
+  mediumSize,
+  mediumMaxSize,
+  smallSize,
+  smallMaxSize,
+  extraSmallSize,
+  extraSmallMaxSize,
+  push,
+}) => html`
+  <form is="vl-form">
+    <div is="vl-form-grid">
+      <div
```

Comment:
Zouden we niet best een label bij deze story zetten? Dat laat ook mooi het gebruik van de form-column zien, nu niet echt.

Reply:
Bedoel je gewoon een label erboven zetten in een form column van 12? Of hoe zou jij het duidelijker maken?

Reply:
Zoals besproken label weglaten.